### PR TITLE
chore(gpu): add tests for N=8192 multi-bit PBS

### PR DESCRIPTION
**Pull request by agnesLeroy**
closes: _please link all relevant issues_

### Check-list:

* [ ] Tests for the changes have been added (for bug fixes / features)
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Relevant issues are marked as resolved/closed, related issues are linked in the description
* [ ] Check for breaking changes (including serialization changes) and add them to commit message following the conventional commit [specification][conventional-breaking]

[conventional-breaking]: https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer

### CODE REVIEW

**Comment by agnesLeroy** [comment]
@IceTDrinker there may be an issue with MULTI_BIT_MESSAGE_3_CARRY_3 parameter sets. I'm trying to add tests on the multi-bit PBS with those parameters but they fail on CPU and GPU.

**Comment by IceTDrinker** [comment]
known issue, some of the algorithms needs power of 2 number of bits cc @tmontaigu he knows that better than me

**Comment by tmontaigu** [comment]
It looks like its core_crypto algorithms that fails not integer ones

**Comment by IceTDrinker** [comment]
yes my bad @tmontaigu Agnes told me that, we'll take a look this afternoon

File: tfhe/src/core_crypto/algorithms/test/mod.rs [hunk 1]
```diff
@@ -197,6 +197,24 @@ pub const MULTI_BIT_2_2_2_PARAMS: MultiBitTestParams<u64> = MultiBitTestParams {
...
+        0.0000003272369292345697,
+    )),
+    decomp_base_log: DecompositionBaseLog(14),
+    decomp_level_count: DecompositionLevelCount(2),
+    glwe_dimension: GlweDimension(1),
+    polynomial_size: PolynomialSize(8192),
+    glwe_noise_distribution: DynamicDistribution::new_gaussian_from_std_dev(StandardDev(
+        0.0000000000000000002168404344971009,
+    )),
+    message_modulus_log: CiphertextModulusLog(8),
```
Comment (IceTDrinker) [inline tfhe/src/core_crypto/algorithms/test/mod.rs:212]: This should be a 6 as 3 bits + 3 bits = 6 bits :)

**Comment by agnesLeroy** [comment]
@IceTDrinker do we want to merge this or should I close the PR instead?

**Comment by pdroalves** [comment]
I rebased this PR on the current main and it looks like the bug was fixed at some point between this PR and today. 3_3 core_crypto tests pass and I can run the primitives for `polynomial_size=8192`.

**Comment by IceTDrinker** [comment]
one of the issue was a wrong setting in the parameters, so maybe that fixed it

**Comment by pdroalves** [comment]
@agnesLeroy can we merge this one?

**Pull request merged**